.gitignore

```diff
@@ -2,6 +2,9 @@
 scripts/test.py
 keys
 
+# gcp token
+token/
```

Comment:
which service account did you use? :-)) do you have the token.json? i would like to keep a copy of it in drive (where we have access)

Reply:
it's called gbq-automate; you can just generate a new token or I can also send it to you